### DISCUSSION TITLE
can i disable Bit-rots feature？

### DISCUSSION THREAD

**Discussion by lddlww** [minio/minio#14272] [Q&A]
is Bit-rots default disable，or not，how can i disable it？

**Comment by klauspost** [comment]
Why would you do that?

**Comment by lddlww** [comment]
i want to compare performance gap between enable bit-rots and disable bit-rots

**Reply by klauspost** [reply]
What do you mean? Bitrot is simply hashing of blocks. It is basically free. It has never shown up on any profiling.

To answer your question, no you cannot disable it.

**Reply by lddlww** [reply]
ok，thanks
